src/jit/emitxarch.cpp

```diff
@@ -3996,15 +4024,15 @@ void emitter::emitIns_AR(instruction ins, emitAttr attr, regNumber base, int off
     emitCurIGsize += sz;
 }
 
-void emitter::emitIns_R_A(instruction ins, emitAttr attr, regNumber reg1, GenTreeIndir* indir, insFormat fmt)
+void emitter::emitIns_R_A(instruction ins, emitAttr attr, regNumber reg1, GenTreeIndir* indir)
```

Comment:
IMO it would be very helpful to have all HW intrinsics emitter functions documented as they play central role HWI implementation.

Reply:
I agree that it would be useful to have these functions documented, in general. But that is a work item for a separate PR.

At least right now, the convention is at least somewhat understandable. `emitIns` emits an instruction and the subsequent modifiers indicate the type and order of the operands that are emitted:
* `A` is an GenTreeIndir address
* `AR` is a register + offset address
* `_C` is a field handle + offset address
* `_R` is a register`
* `_S` is a local variable + offset address

Reply:
The naming convention is separate from this PR and should probably be handled separately as well.

Given that it also impacts more than just the HWIntrinsics, it also probably needs review from additional people.

Reply:
(There is other cleanup that also needs to happen here and which can happen in the other PR, such as changing the regular `emitIns_*` methods to also use the "better" parameter names).